tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_overview.md

```diff
@@ -2,89 +2,217 @@
 
 !---
 
-## Polynomial Fitting
+## Function Approximation
 
-To introduce the concept of FEM, consider a polynomial fitting exercise. When fitting a polynomial
-there is a known set of points as well as a set of coefficients that are unkown for a function
-that has the form:
+To introduce the concept of FEM, consider a polynomial regression exercise. We can
+determine a polynomial function in a:
 
-!equation
-f(x) = a + bx + cx^2 + \dots,
+- +Discrete sense+: Using sampled locations with associated values, we want our polynomial function to be as close to the given data as possible.
+- +Continuous sense+: Using a complicated function, we want to have our polynomial function as close to the given function as possible.
 
-where $a$, $b$ and $c$ are scalar coefficients and $1$, $x$, $x^2$ are "basis functions". Thus, the
-problem is to find $a$, $b$, $c$, etc. such that $f(x)$ passes through the points given.
 
-More generally,
+Let us take the polynomial in the following form:
 
 !equation
-f(x) = \sum_{i=0}^d c_i x^i,
+f(x) = \sum_{i=0}^d c_i x^i
 
-where the $c_i$ are coefficients to be determined. $f(x)$ is unique and interpolary if $d+1$ is the
-same as the number of points needed to fit. This defines a linear system that must be solved to
-find the coefficients.
+where $c_i$ are scalar coefficients (expansion coefficients) and $x^i$ are "basis functions". Thus, the
```

Comment:
```suggestion
where $c_i$ are scalar coefficients (expansion coefficients) and monomials $x^i$ are "basis functions". Thus, the
```

Just to make it clear that, despite the new special terminology, we're still looking at an exponent here rather than an index?